changelog.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased](https://github.com/rokka-io/rokka-client-bundle/compare/1.1.1...master)
 ### Added
 ### Changed
+- Removed usage "dunglas/symfony-lock" from tests
```

Comment:
i would not add this to the changelog, for the consumer of the bundle it is irrelevant and we risk confusing people with it.